pretty_cron/api.py

```diff
@@ -4,21 +4,26 @@
 def prettify_cron(expression):
     pieces = []
     for piece in expression.split(" "):
-        if piece != "*":
+        if piece != "*" and "," not in piece:
             try:
                 piece = int(piece)
             except ValueError:
                 # */2 and other cron expressions aren't supported yet - return
                 # as-is
                 return expression
+        elif "," in piece:
+            splitter = piece.split(",")
+            try:
+                splitter = [int(p) for p in splitter]
+            except ValueError:
+                return expression
+            piece = tuple(splitter)
         pieces.append(piece)
-
```

Comment:
Don't gratuitously change lines in a PR that aren't to do with the functionality you're adding. It's just a stylistic preference, plus it makes your code harder to merge if say these lines get changed before your PR is looked at.